kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerRuleCapabilitiesIntegrationTest.java

```diff
@@ -34,6 +34,7 @@
 import org.kie.server.integrationtests.shared.KieServerSynchronization;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.*;
```

Comment:
In this test class is already imported org.assertj.core.api.Assertions.assertThat so there is no need to import JUnit asserts. Please replace assertEquals(epxpected, actual) with assertThat(actual).isEqualTo(Expected).